### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by RenatoDev3** [vercel/community#4570] [Account Issue]
### Question

Hello, I tried logging into my Vercel account and upon login with Github or any other way I get this message: 
![image](https://github.com/vercel/community/assets/142670217/9240deea-70da-4df3-b877-51a63e36aa62)

I have contacted support over a month ago, searched online for my issue and I can see multiple people have the same issue and it is not being fixed. I haven't gotten a response from the support team in over a month, please fix the issue.

Thank you.

**Comment by amyegan** [comment]
If you let me know the case number, I'll check on the ticket for you 🙇

**Reply by MikeRock51** [reply]
Hi! I have the same issue and here is my case number 00167468. I would appreciate your assistance.

**Reply by RenatoDev3** [reply]
Here it is: 00165149

**Reply by ghost** [reply]
Have same issue, my case: 00163912

**Reply by ravmhmmd** [reply]
Have same issue, my case: 00161213

**Comment by RenatoDev3** [comment]
Hello, @amyegan

After I provided my case number I get a response back from the support team, but once again, over 4 days has passed since the last reply. It definitely shouldn't take this long to fix such a simple issue.

I ask politely that you assist me with the issue at hand.

Thank you!

**Reply by MikeRock51** [reply]
Same here. I just don't get why they've made it this difficult to get an account

**Reply by niruallen** [reply]
Vercel 🤦‍♂️
